_about/notes/0.8.md

```diff
@@ -8,4 +8,47 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model.
+[Learn more]({{home}}/docs/tasks/traffic-management-v1alpha3/)
+
+- **Envoy V2**. Pilot now uses the V2 Envoy API to program the proxies in the mesh, which increases
```

Comment:
Are we officially switching over to v2 by default?

Reply:
i'd like to but i think people lean towards not to have v2 by default.  cc  @costinm